### ISSUE TITLE
Schematic "ng-add" cannot resolve the factory

### ISSUE THREAD

**Issue by mburger81** [kyubisation/angular-server-side-configuration#39]
on **ng add** i'm running into this issue

```
ng add angular-server-side-configuration

Installing packages for tooling via npm.
Installed packages for tooling via npm.
An unhandled exception occurred: Schematic "ng-add" cannot resolve the factory.

 Schematic "ng-add" cannot resolve the factory.
    at NodeModulesEngineHost.createSchematicDescription (/Users/xxxx/node_modules/@angular-devkit/schematics/tools/file-system-engine-host-base.js:175:19)
    at SchematicEngine.createSchematic (/Users/xxxx/node_modules/@angular-devkit/schematics/src/engine/engine.js:219:38)
    at CollectionImpl.createSchematic (/Users/xxxx/node_modules/@angular-devkit/schematics/src/engine/engine.js:69:29)
    at AddCommand.getSchematic (/Users/xxxx/node_modules/@angular/cli/models/schematic-command.js:134:27)
    at AddCommand.runSchematic (/Users/xxxx/node_modules/@angular/cli/models/schematic-command.js:296:32)
    at AddCommand.executeSchematic (/Users/xxxx/node_modules/@angular/cli/commands/add-impl.js:176:31)
    at AddCommand.run (/Users/xxxx/node_modules/@angular/cli/commands/add-impl.js:142:21)
    at async AddCommand.validateAndRun (/Users/xxxx/node_modules/@angular/cli/models/command.js:134:28)
    at async Object.runCommand (/Users/xxxx/node_modules/@angular/cli/models/command-runner.js:201:24)
    at async default_1 (/Users/xxxx/node_modules/@angular/cli/lib/cli/index.js:62:31)
```

**Comment by kyubisation** [comment]
Thanks for reporting this issue.
I have released version 9.0.2, which should solve this issue.
Can you try it out? (You may have to remove angular-server-side-configuration from your package.json first.)

**Comment by kyubisation** [comment]
Closing this issue due to inactivity.
Feel free to reopen the issue, if there is still a problem.
